kubespawner/templates/style.css

```diff
@@ -27,3 +27,7 @@
   margin-right: 8px;
   min-width: 96px;
 }
+
+.hide {
```

Comment:
bootstrap 3, which is used here, has a .hidden that we can use instead https://getbootstrap.com/docs/3.4/css/#helper-classes